server/presenters/document.ts

```diff
@@ -25,80 +25,58 @@ async function replaceImageAttachments(text: string) {
 }
 
 export default async function present(
-  document: Document,
+  document: any,
```

Comment:
Until the backend models are typed this is honestly better than a file packed full of `ts-expect-error`